frog/imports/client/Wiki/index.js

```diff
@@ -520,6 +595,82 @@ class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
     return { liId, pageId };
   };
 
+  editAccess = async action => {
+    if (this.state.isOwner) return true;
+    if (action === 'createPage') {
+      if (!this.state.settings?.allowPageCreation) {
+        this.props.showModal(
```

Comment:
This Modal should be moved to its own file. This indicates we might need a standard way to display error messages

Reply:
Yes, we do need a standard way to display error/success messages, but in absence of such a method we would need to resort to modals and creating a file for 10 lines of code seems too much to me.